frontend/simple/views/SignUp.vue

```diff
@@ -7,18 +7,16 @@
      .block      base/classes.sass (just adds 20px margin-bottom except for last)
      -->
     <form novalidate ref="form"
-      name="formData" class="container signup"
+      name="formData" class="container"
       @submit.prevent="submit"
     >
-      <div class="box centered" style="max-width:400px">
+      <div class="box mx-auto" style="max-width: 400px">
         <div class="level is-mobile">
-          <div class="level-left">
-            <div class="level-item">
-              <p class="subtitle"><i18n>Sign Up</i18n></p>
-            </div>
+          <div class="level-left mt-md">
+            <p class="subtitle"><i18n>Sign Up</i18n></p>
           </div>
           <div class="level-right">
-            <p class="level-item content is-small">
+            <p class="mt-md content is-small">
```

Comment:
For some reason `level-item` has been removed here and in many other places.

It makes sense for us to stick with `.level-item`s as level items go into the level.